### ISSUE TITLE
adverse data

### ISSUE THREAD

**Issue by malice960** [WaterScenes/WaterScenes#15]
Thank you for your excellent work. I have a question. In Table 5 of the paper, did the test data for advance_weather and advance_lighting in BENCHMARK Results of Object Detection filter all the images corresponding to the train, val, and test datasets? Does this mean that the images in the training dataset will be tested during testing, or only the corresponding parts from the test dataset will be filtered? Looking forward to your reply

**Comment by yaoshanliang** [comment]
@malice960 ONLY the adverse images in the test set will be used for testing. Thank you.

**Comment by malice960** [comment]
get it! thank you!
